utils/build.ps1

```diff
@@ -2881,8 +2880,10 @@ if (-not $SkipBuild -and $Allocator -eq "mimalloc") {
   Invoke-BuildStep Build-Mimalloc $HostArch
 }
 
-if (-not $SkipBuild -and -not $IsCrossCompiling) {
+if (-not $SkipBuild) {
   Invoke-BuildStep Build-Inspect $HostArch
```

Comment:
I think that we can move this into the build step blocks above.

Reply:
Done

Reply:
I think that you forgot to push the change.

Reply:
Sorry! PTAL

Reply:
Sorry for the latency. I now see the new change uploaded.